ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -62,7 +69,11 @@ internal fun Template2(state: PaywallViewState.Loaded, viewModel: PaywallViewMod
 }
 
 @Composable
-private fun Template2MainContent(state: PaywallViewState.Loaded, viewModel: PaywallViewModel) {
+private fun Template2MainContent(
+    state: PaywallViewState.Loaded,
+    viewModel: PaywallViewModel,
+    childModifier: Modifier,
```

Comment:
I wonder if we should rename to `contentModifier` or `loadingContentModifier` so it's clearer.

Reply:
👍🏻